Kalyani/HTML/Assginment1/Assin4.html

```diff
@@ -1,7 +1,14 @@
 <!DOCTYPE html>
 <html>
-<head>    
-<title> <h2>Introduction</h2></title>
-<p>
-I am Kalyani Deshpande<br>
-I am a Software Enginner
+	<head>
+	    <title>
+	        <h2>Introduction</h2>
+	    </title>
+	</head>
+	<body style="background-color: lightblue">
+	    <p>
+	        <i><h2 style="color: red">I am Kalyani Deshpande</h2></i><br />I am a Software Enginner.
```

Comment:
<i> is inline element and <h2> is block level element u can not include block level elements inside inline element.so do the opposite of what u have done here